### DISCUSSION TITLE
Logging

### DISCUSSION THREAD

**Discussion by clmarshall** [cline/cline#1066] [Feature Requests]
There is obviously a lot of back and forth going on between roo cline and the llm.  I'd like a log of that conversation so that I can refer back to it.  This will help me improve my custom instructions and prompts over time.  I do not believe this should be turned on by default, but it is valuable to have as a resource.

**Comment by MBrekhof** [comment]
just curious, what is different from https://github.com/cline/cline/discussions/786 ?
